rpc/client.go

```diff
@@ -16,13 +16,18 @@ import (
 // They return information related to auctions, bids and validators.
 // This information is necessary for users involved with node operations and validation.
 type ClientPOS interface {
-	// GetLatestAuctionInfo returns the types.ValidatorBid and types.EraValidators from the most recent Block.
-	GetLatestAuctionInfo(ctx context.Context) (StateGetAuctionInfoResult, error)
-	// GetAuctionInfoByHash returns the types.ValidatorBid and types.EraValidators of either a specific Block by hash
-	GetAuctionInfoByHash(ctx context.Context, blockHash string) (StateGetAuctionInfoResult, error)
-	// GetAuctionInfoByHeight returns the types.ValidatorBid and types.EraValidators of either a specific Block by height
-	GetAuctionInfoByHeight(ctx context.Context, height uint64) (StateGetAuctionInfoResult, error)
-
+	// GetLatestAuctionInfoV1 returns the types.ValidatorBid and types.EraValidators from the most recent Block.
+	GetLatestAuctionInfoV1(ctx context.Context) (StateGetAuctionInfoV1Result, error)
+	// GetAuctionInfoV1ByHash returns the types.ValidatorBid and types.EraValidators of either a specific Block by hash
+	GetAuctionInfoV1ByHash(ctx context.Context, blockHash string) (StateGetAuctionInfoV1Result, error)
+	// GetAuctionInfoV1ByHeight returns the types.ValidatorBid and types.EraValidators of either a specific Block by height
+	GetAuctionInfoV1ByHeight(ctx context.Context, height uint64) (StateGetAuctionInfoV1Result, error)
+	// GetLatestAuctionInfoV2 returns the types.ValidatorBid and types.EraValidators from the most recent Block.
+	GetLatestAuctionInfoV2(ctx context.Context) (StateGetAuctionInfoV2Result, error)
+	// GetAuctionInfoV2ByHash returns the types.ValidatorBid and types.EraValidators of either a specific Block by hash
+	GetAuctionInfoV2ByHash(ctx context.Context, blockHash string) (StateGetAuctionInfoV2Result, error)
+	// GetAuctionInfoV2ByHeight returns the types.ValidatorBid and types.EraValidators of either a specific Block by height
+	GetAuctionInfoV2ByHeight(ctx context.Context, height uint64) (StateGetAuctionInfoV2Result, error)
```

Comment:
Let's add the following aliases for V2:
- `GetLatestAuctionInfo`
- `GetAuctionInfoByHash`
- `GetAuctionInfoByHeight`